common/types/hashable.go

```diff
@@ -9,4 +9,5 @@ type Hashable interface {
 	comparable
 
 	Bytes() []byte
+	Hex() string
```

Comment:
why is this needed?

for `common.Address` `Hex()` and `String()` do the same thing - and hex encoding is a bit more of an EVM concept

Reply:
Thanks for pointing that out. My Mistake, as i thought that `Hex()` encoding is a generalised concept. 

I added `Hex()` because it was being called by the logger in `eth_confirmer.go`.

However it looks like `String()` for common.hash also calls `Hex()`, so it makes sense for us to use `String()`. 

Made the following changes:
1. Removed the `Hex()` from the interface. 
2. Used `String()` instead of `Hex()` for `eth_confirmer.go` log.